crowbar_framework/app/models/service_object.rb

```diff
@@ -1551,26 +1544,6 @@ def run_remote_chef_client(node, command, logfile_name)
     }
   end
 
-  def chef_daemon(action, node_list)
```

Comment:
@dirkmueller @MaximilianMeister introduced them specifically for this purpose (in the same commit, even).  I just checked and can't see anything else using them.  But it doesn't matter too much if a few unused methods are left around in old stable branches.  The important thing is not to do that in `master`.